demo/README.md

```diff
@@ -225,6 +225,7 @@ Optional arguments:
 - `INPUT_STEP`: Input step for sampling frames, which can help to get more spare input. If not specified , it will be set to 1.
```

Comment:
Here

Reply:
Nope, they are not the same. For example, if set `input_step` as 4, for every consecutive 4 frames, we randomly append one frame to the frame queue and drop the other three frames. However, getting predictions every 4 frames means you first get the prediction for frame 0-64, the 4-68, etc. 
The input-size might works for TSN, but not a general solution to both 2D and 3D approaches.